libp2p/wire.nim

```diff
@@ -71,7 +71,7 @@ proc connect*(ma: MultiAddress, bufferSize = DefaultStreamBufferSize,
   ## Open new connection to remote peer with address ``ma`` and create
   ## new transport object ``StreamTransport`` for established connection.
   ## ``bufferSize`` is size of internal buffer for transport.
-  if not(RTRANSPMA.match(ma)):
+  if not(RTRANSPMA.matchPartial(ma)):
```

Comment:
What the purpose of doing this change?

Reply:
See above answer.
Any implications about this choice btw? It definitely fixes the issue but yeah I'm happy to discuss it. 
It used to be like this before client side.